engine/src/flutter/shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -447,6 +447,49 @@ FLUTTER_DARWIN_EXPORT
 
 @end
 
+/**
+ * Exposes parts of a `FlutterEngine` for registration purposes.
+ *
+ * This is used when the engine is created implicitly to allow registering
+ * plugins, application-level method channels, platform views, etc.
+ */
+@interface FlutterImplicitEngineBridge : NSObject
```

Comment:
Sorry, one final small change request, since I've just been in the world of trying to replace OCMock with fakes: let's make this a protocol instead of a class, and then have a private class (or just the engine itself) implement it. That makes calling it from tests much easier.